### ISSUE TITLE
stdout/stdin

### ISSUE THREAD

**Issue by rick-monster** [monome/libavr32#3]
is it easy to enable regular old blocking stdout/stdin on the serial port?  Think this may be the quickest route to start experimenting with miniPicoLisp on aleph...

**Comment by catfact** [comment]
yeah, the usart driver has both blocking and asynchronous i/o functions ( [ `usart_putchar`, `usart_getchar`], [`usart_write_char`, `usart_read_char`])
https://github.com/monome/libavr32/blob/master/asf/avr32/drivers/usart/usart.c#L851

is that what you mean?

**Comment by rick-monster** [comment]
I've got something cooking here - found an old version of picolisp with none of the crazy preprocessor malarky for building bits of lisp into the rom - just 5 c files.  And think I've got the thing more or less working.  However if I initialise stdout before I kick off the lisp things blow up (trying to figure out where exactly).

I would like to try defining stdin/stdout using fdevopen, rather than the in-memory stream.  See if that works better.  However fdevopen seems to not work in an an aleph project, which is confusing me somewhat.  Just getting flamed in #avr irc right now trying to figure out what's up...
